order/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+class OrderItem(models.Model):
+    product_id = models.ForeignKey('product.Product', on_delete=models.CASCADE)
+    order_id = models.ForeignKey('Order', on_delete=models.CASCADE)
+    order_item_quantity = models.IntegerField()
+    order_item_price = models.CharField(max_length=80)
+    order_shipment = models.CharField(max_length=100)
```

Comment:
class Shipment(models.Model):
        company_name = ..
        tracking_number = ...
        status = ...(배송중, 배송완료 등..)

```suggestion
    shipment = models.ForeignKey('Shipment')
```

Reply:
넵, 수정했습니다!